src/win32_window.c

```diff
@@ -1436,6 +1458,10 @@ static int createNativeWindow(_GLFWwindow* window,
 
     _glfwGetWindowSizeWin32(window, &window->win32.width, &window->win32.height);
 
+    if (window->theme.variation == GLFW_THEME_DEFAULT) {
+        applySystemTheme(window->win32.handle);
+    }
+
```

Comment:
There's no need check this variable. It's solely to prevent the user from having to manage the memory of theme instances returned with glfwGetTheme.

Reply:
So, always use the system default when creating a window.